jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -0,0 +1,221 @@
+/*[INCLUDE-IF JAVA_SPEC_VERSION >= 19]*/
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package jdk.internal.vm;
+
+import java.util.Set;
+import java.util.function.Supplier;
+
+/**
+ * Continuation class performing the mount/unmount operation for VirtualThread
+ */
+public class Continuation {
+	long vmref; /* J9VMContinuation */
```

Comment:
```suggestion
	long vmRef; /* J9VMContinuation */
```
This is consistent w/ `j.l.Class.vmRef`and `j.l. ClassLoader.vmRef`, pls update `j.l. InternalConstantPool.vmref` as well.